network/network_windows.go

```diff
@@ -337,6 +397,13 @@ func (nm *networkManager) newNetworkImplHnsV2(nwInfo *NetworkInfo, extIf *extern
 			}
 
 			log.Printf("[net] Successfully created hcn network with response: %+v", hnsResponse)
+
+			// only add net rules if it's dualStackOverlay mode and hnsNetwork is created at first time
+			if util.DualStackOverlay == DualStackOverlay {
```

Comment:
trying to understand the significance of this check.. isn't these comparing 2 constant variables?